src/Wallabag/CoreBundle/Resources/views/themes/material/Entry/entry.html.twig

```diff
@@ -216,6 +216,14 @@
         </li>
         {% endif %}
 
+        <li class="bold border-top border-bottom hide-on-med-and-down">
+            <a class="waves-effect collapsible-header" title="{{ 'menu.top.random_entry'|trans }}" href="{{ path('random_entry', { 'type': 'all' }) }}">
+                <i class="material-icons small">casino</i>
+                <span>{{ 'menu.top.random_entry'|trans }}</span>
+            </a>
+            <div class="collapsible-body"></div>
```

Comment:
Note : All other items have this `<div>` but I'm not even sure it's needed in this case.